ios/brave-ios/Sources/Brave/Frontend/Browser/New Tab Page/Backgrounds/NewTabPageBackgroundButtonsView.swift

```diff
@@ -221,4 +279,43 @@ extension NewTabPageBackgroundButtonsView {
       layer.shadowPath = UIBezierPath(ovalIn: bounds).cgPath
     }
   }
+  private class PlayButton: SpringButton {
+    let imageView = UIImageView(
+      image: UIImage(braveSystemNamed: "leo.play.circle", compatibleWith: nil)!
+    ).then {
+      $0.tintColor = .white
+      $0.contentMode = .scaleAspectFit
+    }
+
+    private let backgroundView = UIVisualEffectView(
+      effect: UIBlurEffect(style: .systemUltraThinMaterialDark)
+    ).then {
+      $0.clipsToBounds = true
+      $0.isUserInteractionEnabled = false
+    }
+
+    override init(frame: CGRect) {
+      super.init(frame: frame)
+
+      clipsToBounds = true
+
+      addSubview(backgroundView)
+
+      backgroundView.contentView.addSubview(imageView)
+
+      backgroundView.snp.makeConstraints {
+        $0.edges.equalToSuperview()
+      }
+      imageView.snp.makeConstraints {
+        $0.width.equalTo(40)
+        $0.height.equalTo(40)
```

Comment:
This is incorrect for sizing SF Symbols, you need to apply a large size to the image itself. Please see https://developer.apple.com/documentation/uikit/uiimage/symbolconfiguration or find usages of `applyingSymbolConfiguration` in the codebase